apps/examples/smartfs/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -175,16 +175,26 @@ static int write_text_file(int fd, size_t bufsize)
 	return ret;
 }
 
-static int create_file(char *path, ssize_t bufsize, file_type_t type)
+static int create_file(char *path, ssize_t bufsize, file_type_t type, bool is_backup_file)
```

Comment:
Do you think it is better to add the type of backup file using enums(small random character, small text file, large random char) for which the size is being checked and display the log accordingly ??
Otherwise it will be tough to make out which one failed..